src/odemis/gui/conf/data.py

```diff
@@ -235,6 +239,52 @@
                 "control_type": odemis.gui.CONTROL_NONE,
             }),
         )),
+        "ion-beam":
+        OrderedDict((
+            ("accelVoltage", {
+                "label": "Accel. Voltage",
+                "tooltip": "Accelerating voltage",
+                "event": wx.EVT_SCROLL_CHANGED  # only affects when it's a slider
+            }),
+            ("probeCurrent", {
+                "label": "Beam Current",
+                "control_type": odemis.gui.CONTROL_SLIDER,
+                "type": "float",
+                "scale": "linear",
+                "event": wx.EVT_SCROLL_CHANGED
+            }),
+            ("resolution", {
+                "label": "Resolution",
+                "control_type": odemis.gui.CONTROL_COMBO,
+                "tooltip": "Number of pixels in the image",
+                "choices": None,
```

Comment:
for the e-beam the resolution choices are `"choices": util.resolution_from_range,`, why not have that here?

Reply:
this resolution is set directly, so it has choices. so it can be used directly. whereas i think the other resolutions are set indirectly from scale / pixelsize

Reply:
makes sense